CONTRIBUTING.md

```diff
@@ -1,107 +1,4 @@
-
-### 1) Fork on GitHub
-
-Fork the appropriate repository by clicking the Fork button (top right) on GitHub.
-
-### 2) Create a Local Fork
-
-From whatever directory you want to have this code, clone this repository and setup some sane defaults:
-
-```
-$ git clone https://github.com/$user/chef-atlantis/
-# if you are a collaborator you can use the @git notation
-
-$ cd chef-catlantis
-
-$ git remote add upstream https://github.com/majormoses/chef-atlantis.git
-# if you are a collaborator you can use the @git notation
-
-# Never allow a push to upstream master
-$ git remote set-url --push upstream no_push
-
-# Confirm that your remotes make sense:
-$ git remote -v
-```
-
-### 3) Create a Branch for Your Contribution
-
-Begin by updating your local fork of the cookbook:
-
-```
-$ git fetch upstream
-$ git checkout master
-$ git rebase upstream/master
-```
-
-Create a new, descriptively named branch to contain your change:
-
-```
-$ git checkout -b feature/myfeature
-```
-
-Now hack away at your awesome feature on the `feature/myfeature` branch.
-
-### 4) Testing Your Code
-
-Please see [Our testing documentation](TESTING.md)
-
-### 5) Committing Code
-
-Commit your changes with a thoughtful commit message.
-
-```
-$ git commit -am "Adding a feature that does Foo
-
-Foo is a particularly helpful status when working with Bar. Designed to gather XYZ from the foobar interface."
-```
-
-Repeat the commit process as often as you need and then edit/test/repeat. Minor edits can be added to your last commit quite easily:
-
-```
-$ git add -u
-$ git commit --amend
-```
-
-### 6) Pushing to GitHub
-
-When ready to review (or just to establish an offsite backup or your work), push your branch to your fork on GitHub:
-
-```
-$ git push origin feature/myfeature
-```
-
-If you recently used `commit --amend`, you may need to force push:
-
-```
-$ git push -f origin feature/myfeature
-```
-
-### 7) Create a Pull Request
-
-Create a pull request by visiting https://github.com/majormoses/chef-atlantis/ and following the instructions at the top of the screen.
-
-After the PR is submitted, project maintainers will review it.
-
-### 8) Responding to your Pull Request
-
-PRs are rarely merged without some discussion with a maintainer. This is to ensure the larger Chef community benefits from all code contributions.
-
-They will use a system of labels, like `Status: Awaiting Response`, to indicate they need your feedback. Please regularly check your open PRs, easily found at https://github.com/pulls, to help maintainer's get the information needed to merge your code.
-
-## Thank You
-
-We :heart: your participation and appreciate the unique perspective you bring to our community.
+Please refer to
+[https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/CONTRIBUTING.MD](https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/CONTRIBUTING.MD)
```

Comment:
DRY a lot more sense when its part of a larger organization than a random cookbook :wink: